solidity/contracts/deposit/DepositLiquidation.sol

```diff
@@ -61,42 +61,37 @@ library DepositLiquidation {
         return (_bondValue.mul(100).div(_lotValue));
     }
 
-    /// @notice         Starts signer liquidation due to fraud.
-    /// @dev            Liquidation is done by auction.
-    /// @param  _d      Deposit storage pointer.
-    function startSignerFraudLiquidation(DepositUtils.Deposit storage _d) internal {
-        _d.logStartedLiquidation(true);
+    /// @notice           Starts signer liquidation due to abort, undercollateralization, or fraud
```

Comment:
How about something like “Starts signer liquidation by seizing signer bonds. If the deposit is currently being redeemed, the redeemer receives the full bond value; otherwise, a falling price auction begins to buy 1 TBTC in exchange for a portion of the seized bonds; see purchaseSignerBondsAtAuction.”

Since this is internal, I think the whole thing can be `@dev`.